backend/routes/time.js

```diff
@@ -12,49 +12,53 @@ const processInput = require('../helpers/processInput');
 const timeQueries = require('../queries/time');
 
 //Gets the number of hours banked by a volunteer
-router.get('/past/:v_id', async (req, res, next) => {
+router.get('/hours/:v_id', async (req, res, next) => {
     try {
         const vId = processInput(req.params.v_id, "idNum", "volunteer id");
         let allBankedHours = await timeQueries.allBankedHours(vId);
 
-        res.json({
-            payload: allBankedHours,
-            message: "Success",
-            err: false
-        });
+        res.status(200)
+            .json({
+                payload: allBankedHours,
+                message: "Success",
+                err: false
+            });
     } catch (err) {
         handleError(err, req, res, next);
     }
 });
 
-//Gets the number of hours planned by a volunteer
-router.get('/upcoming/:v_id', async (req, res, next) => {
-    try {
-        const vId = processInput(req.params.v_id, "idNum", "volunteer id");
-        let allPlannedHours = await timeQueries.allPlannedHours(vId);
-        res.json({
-            payload: allPlannedHours,
-            message: "Success",
-            err: false
-        });
-    } catch (err) {
-        handleError(err, req, res, next);
-    }
-});
 
 //Gets the number of hours by all volunteers
 router.get('/', async (req, res, next) => {
     try {
         let allHours = await timeQueries.allHours();
 
-        res.json({
-            payload: allHours,
-            message: "Success",
-            err: false
-        });
+        res.status(200)
+            .json({
+                payload: allHours,
+                message: "Success",
+                err: false
+            });
     } catch (err) {
         handleError(err, req, res, next);
     }
 });
 
+router.patch('/update', async (req, res, next) => {
+    try {
```

Comment:
we need to protect this patch by checking if the user whom doing the patch is an admin user
`try {
if (req.user && req.user.admin) {
// .... rest of the code
} else {
throw new Error('403__Not allowed to perform this operation')
} catch ... rest of the code`

Reply:
The route is protected now.